tests/test_data/doc-example-collections.json

```diff
@@ -198,7 +198,7 @@
                     },
                     "measurementType": {
                         "method": "mean",
-                        "period": "-PT10M/PT0M"
+                        "duration": "-PT10M/PT0M"
```

Comment:
Thanks for looking at this!

The examples were changed to not have this strange `duration/duration` convention anymore, see:
https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/pull/577/commits/4e82424c1f8c0901222fdb89a4bb6f471ef16b8f

And also the examples in the 1.2 draft version:
https://docs.ogc.org/DRAFTS/19-086r7.html#measurementType_methods_desc

This was also brought up by Paul in the original Issue:
https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/issues/560

Could you make this changes in this example?